packages/core/src/mutants/MutantTestMatcher.ts

```diff
@@ -1,218 +1,218 @@
-   * @returns The MatchedMutant
-   */
-  private mapMutantOnMatchedMutant(testableMutant: TestableMutant): MatchedMutant {
-    const matchedMutant: MatchedMutant = {
-      fileName: testableMutant.mutant.fileName,
-      id: testableMutant.id,
-      mutatorName: testableMutant.mutant.mutatorName,
-      replacement: testableMutant.mutant.replacement,
-      runAllTests: testableMutant.runAllTests,
-      scopedTestIds: testableMutant.selectedTests.map((testSelection) => testSelection.id),
-      timeSpentScopedTests: testableMutant.timeSpentScopedTests,
-    };
-    return Object.freeze(matchedMutant);
-  }
-
-  private findMatchingStatement(location: LocationHelper, fileCoverage: CoverageMaps): StatementIndex | null {
-    const statementIndex = this.findMatchingStatementInMap(location, fileCoverage.statementMap);
-    if (statementIndex) {
-      return {
-        index: statementIndex,
-        kind: StatementIndexKind.Statement,
-      };
-    } else {
-      const functionIndex = this.findMatchingStatementInMap(location, fileCoverage.fnMap);
-      if (functionIndex) {
-        return {
-          index: functionIndex,
-          kind: StatementIndexKind.Function,
-        };
-      } else {
-        return null;
-      }
-    }
-  }
-
-  /**
-   * Finds the smallest statement that covers a location
-   * @param needle The location to find.
-   * @param haystack the statement map or function map to search in.
-   * @returns The index of the smallest statement surrounding the location, or null if not found.
-   */
-  private findMatchingStatementInMap(needle: LocationHelper, haystack: StatementMap): string | null {
-    let smallestStatement: { index: string | null; location: LocationHelper } = {
-      index: null,
-      location: LocationHelper.MAX_VALUE,
-    };
-    if (haystack) {
-      Object.keys(haystack).forEach((statementId) => {
-        const statementLocation = haystack[statementId];
-
-        if (needle.isCoveredBy(statementLocation) && smallestStatement.location.isSmallerArea(statementLocation)) {
-          smallestStatement = {
-            index: statementId,
-            location: new LocationHelper(statementLocation),
-          };
-        }
-      });
-    }
-    return smallestStatement.index;
-  }
-
-  private findCoverageCollectionForTest(testId: number): CoverageCollection | null {
-    if (this.initialRunResult.runResult.coverage) {
-      if (this.isCoveragePerTestResult(this.initialRunResult.runResult.coverage)) {
-        return this.initialRunResult.runResult.coverage.deviations[testId];
-      } else {
-        return this.initialRunResult.runResult.coverage;
-      }
-    } else {
-      return null;
-    }
-  }
-
-  private isCoveragePerTestResult(_coverage: CoverageCollection | CoveragePerTestResult | undefined): _coverage is CoveragePerTestResult {
-    return this.options.coverageAnalysis === 'perTest';
-  }
-}
+// import { Mutant, StrykerOptions } from '@stryker-mutator/api/core';
```

Comment:
commented file?

Reply:
Yeah, one of the ones that need to go. Probably best if I remove it indeed. I do want to keep some of it for now. For example, the new `Checker` code can be loosely based on the `Transpiler` code.

Reply:
Although I think it is better to remove it, since we could forget about it later, ok. We just need to keep it in mind :)

Reply:
No, you're right. Classes that are already refactored/replaced should be removed.

Reply:
Done, all dead code is gone now.